### ISSUE TITLE
More features?

### ISSUE THREAD

**Issue by TWINGSISTER** [CodyAdam/tool__numworks-calc#1]
It would be nice to have a display +  keyboard full screen version like in the original version, or better a display + a (larger) keyboard without display.
A giant leap forward would be to have the save/restore from a local file.
Probably this could be implemented saving the status of the WHOLE computation from Electron and this wouldn't be that long to implement.
How one can rebuild this in Windows?

**Comment by codyadam** [comment]
> It would be nice to have a display + keyboard full screen version like in the original version, or better a display + a (larger) keyboard without display.

I agree, but it depends on the user needs. I locked the current window format so that the calculator is in full mode all the time. My goal was to replace the non-functional Windows calculator. You are free to unlock the window for your needs.

> A giant leap forward would be to have the save/restore from a local file.
> Probably this could be implemented saving the status of the WHOLE computation from Electron and this wouldn't be that long to implement.

Indeed, but I don't think it is really worth the time. I don't often find the need for a history of previous sessions.

> How one can rebuild this in Windows?

This is a very simple Electron wrapper around the simulator which make it very easy to edit.

Install node dependencies : 
```npm install```

Run the electron app :
```npm run start```

Build to executable :
```npm run dist```
